### PR TITLE
Add Flask Cheat Sheet

### PR DESCRIPTION
**Pull request by Tigonchen**
## What does this PR do?
Add resource

### Description

Flask Cheat Sheet
HTML

### Why is this valuable (or not)?

FREE

## Checklist:
- [x] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md).
- [x] [Search](https://ebookfoundation.github.io/free-programming-books-search/) for duplicates.
- [x] Include author(s) and platform where appropriate.
- [x] Put lists in alphabetical order, correct spacing.
- [x] Add needed indications (PDF, access notes, under construction).
- [x] Used an informative name for this pull request.

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

**Comment by LuigiImVector** [comment]
fix linter errors

File: more/free-programming-cheatsheets.md [hunk 1]
```diff
@@ -325,6 +325,7 @@
 #### Flask
 
 * [Flask Cheatsheet](https://www.codewithharry.com/blogpost/flask-cheatsheet/) - Code with Harry (HTML)
+* [Flask Cheat Sheet](https://codeinsightacademy.com/blog/python/flask-cheat-sheet/) - Code Insight Academy (HTML)
```
Comment (bugbunnyy) [inline more/free-programming-cheatsheets.md:328]: swap this and above line to solve linter warning.

Comment (shrn01) [inline more/free-programming-cheatsheets.md:328]:
```suggestion
* [Flask Cheatsheet](https://www.codewithharry.com/blogpost/flask-cheatsheet/) - Code with Harry (HTML)
```

File: more/free-programming-cheatsheets.md [hunk 2]
```diff
@@ -325,6 +325,7 @@
 #### Flask
 
 * [Flask Cheatsheet](https://www.codewithharry.com/blogpost/flask-cheatsheet/) - Code with Harry (HTML)
```
Comment (shrn01) [inline more/free-programming-cheatsheets.md:327]:
```suggestion
* [Flask Cheat Sheet](https://codeinsightacademy.com/blog/python/flask-cheat-sheet) - Code Insight Academy (HTML)
```

**Pull request merged**